ground_truth_labeling_jobs/bring_your_own_model_for_sagemaker_labeling_workflows_with_active_learning/bring_your_own_model_for_sagemaker_labeling_workflows_with_active_learning.ipynb

```diff
@@ -0,0 +1,715 @@
+{
```

Comment:
<p>Maybe say "The next cell will  update the value of <em>tokenizer_bucket</em> in our training and prediction scripts." That way the reader doesn't think they have to do it manually.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1121/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>